.github/workflows/deprecated.yml

```diff
@@ -0,0 +1,30 @@
+name: Find deprecated softwares
+
+on:
+  schedule:
+    - cron: '0 20 * * 1'
+
+jobs:
+  black:
+    name: Find deprecated softwares
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
+      - name: Install toml python lib
+        run: |
+          pip3 install toml tomlkit gitpython
+
```

Comment:
From contributors meeting: we need to actually run the script somewhere around here :p